docs/source/install.rst

```diff
@@ -4,6 +4,89 @@
 Installing Volttron
 ===================
 
+Installing Linux Virtual Machine
+--------------------------------
+Volttron requires a Linux system to run. For most this will require virtual machine (VM). This section will describe the
+steps necessary to install VOLTTRON using Oracle VirtualBox software. Virtual Box is free and can be
+downloaded from https://www.virtualbox.org/wiki/Downloads.
+
+|VirtualBox Download|
+
+.. |VirtualBox Download| image:: images/vbox-download.png
+
+After installing VirtualBox, it will also be necessary to download a Linux operating system image for use on the VM.
+Ubuntu 14.04 LTS or Linux Mint 17 or later is the recommended Linux operating system for use with VOLTTRON, however any
+distribution will work. Linux Mint 18.3 with the Xfce desktop is used as an example, however platform setup in Ubuntu
+should be identical. A 32-bit version of Linux should be used when running VOLTTRON on a system with limited hardware
```

Comment:
For this example we are using a Linux Mint 18.3 version  with Xfce desktop.

Make the 32 bit version a note not inline with everything else

A note is a construct in the rtf format...i.e. a bubble or an aside.